var/spack/repos/builtin/packages/openssl/package.py

```diff
@@ -61,6 +63,8 @@ class Openssl(Package):
     version('1.0.1r', '1abd905e079542ccae948af37e393d28')
     version('1.0.1h', '8d6d684a9430d5cc98a62a5d8fbda8cf')
 
+    variant('system-certs', default=True, description='Use system certificates')
```

Comment:
Can you remove the `-` from the variant name here?  Technically it is allowed but we decided to prefer not to use it.  use `systemcerts`

Reply:
Done. :-)